eng/Packages.Data.props

```diff
@@ -74,7 +74,7 @@
 
     <!-- Azure SDK packages -->
     <PackageReference Update="Azure.Core" Version="1.12.0" />
-    <PackageReference Update="Azure.Core.Amqp" Version="1.0.0" />
+    <PackageReference Update="Azure.Core.Amqp" Version="1.1.0-beta.1" Condition="'$(HasReleaseVersion)' == 'false'"/>
```

Comment:
I'm not familiar with build var.   What's it do?

Reply:
This should prevent any GA packages from using this dependency.

Reply:
Interesting idea. We should consider this for Azure.Core.Experimental as well.

cc @pakrym